pkg/controller/elasticsearch/settings/fields.go

```diff
@@ -19,6 +19,7 @@ const (
 	PathData = "path.data"
 	PathLogs = "path.logs"
 
+	XPackSecurityAuthcRealmsFileFile1Order          = "xpack.security.authc.realms.file.file1.order"
```

Comment:
I am a bit confused by this because my understanding of the [documentation](https://www.elastic.co/guide/en/elastic-stack-overview/current/realm-chains.html) suggests that this should look like the following:

```
xpack.security.authc.realms.file1.type = file
xpack.security.authc.realms.file1.order = -100
```

I see from the issue that Jordan's working configuration looks like what you have here, so I am probably mistaken. I just wanted to point it out just in case.

Reply:
Good point! I'm using the syntax introduced in [7.x](30241) that will not work in 6.8 and which is a [breaking change](https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking-changes-7.0.html#include-realm-type-in-setting).

I'm going to support the 2 different syntaxes given the version.